http.go

```diff
@@ -0,0 +1,68 @@
+package discordgo
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+)
+
+type WebhookServer struct {
```

Comment:
Not sure if that's a good idea to call this a `WebhookServer` if it serves only interactions.
I would suggest to rename it to something like `InteractionsWebhookServer` or `InteractionsHTTPServer`